bin/key-mapper-gtk

```diff
@@ -24,13 +24,25 @@
 
 import sys
 import atexit
+import gettext
+import locale
+from keymapper.data import get_data_path
+import os.path
 from argparse import ArgumentParser
 
 import gi
 gi.require_version('Gtk', '3.0')
 gi.require_version('GLib', '2.0')
 from gi.repository import Gtk
 
+APP_NAME = 'key-mapper'
+LOCALE_DIR = os.path.join(get_data_path(), 'lang')
+
+locale.bindtextdomain(APP_NAME, LOCALE_DIR)
+locale.textdomain(APP_NAME)
+
+print('Using locale directory: {}'.format(LOCALE_DIR))
```

Comment:
Can you use the logger module instead of print here? 

`logger.debug('Using locale directory: %s', LOCALE_DIR)`